test/code/plugins/squid_log_parser_plugintest.rb

```diff
@@ -0,0 +1,61 @@
+## squid_log_parser_plugintest
+## Created by Alessandro Cardoso
+## 
+module Fluent
```

Comment:
You should remove the line "module Fluent". Unit tests do not need to be in this module. I believe this is why Travis is failing.